website/public/css/tasks.styl

```diff
@@ -349,7 +350,7 @@ for $stage in $stages
     opacity: 0.75 !important
 
   // completed icon
-  input[type=checkbox]:checked + label:after
+  input[type=checkbox]:checked + label:after, input[type=checkbox]:focus + label:after
```

Comment:
Very clever, and close to what I want. Let's apply the same stylings as if it's being hovered over (slightly different background and checkmark color). In addition, that same styling should apply to it when it's gray, so you can tell when you've selected it once it's already completed.